### DISCUSSION TITLE
Port onenord  theme to Warp Terminal

### DISCUSSION THREAD

**Discussion by PrashanthShetty** [rmehri01/onenord.nvim#71] [General]
Hello,
First of all thank you very much for the "Onenord" Theme.  One of the the best themes I have experienced and enjoying it. Is it possible to port the theme to Warp Terminal? 

Regards,
PS.

**Comment by rmehri01** [comment]
Thanks, I'm glad you like the theme! I've just added basic version since I don't use Warp, feel free to change it as you see fit: https://github.com/rmehri01/onenord.nvim/blob/main/extras/warp/onenord.yaml

**Reply by PrashanthShetty** [reply]
Thank you very much for the quick response and solution. It works and looks great.
